roles/taca/templates/taca_fastq_delivery.yml.j2

```diff
@@ -0,0 +1,21 @@
+log:
+    file: "{{ ngi_pipeline_logs}}/taca.log"
+
+deliver:
+    rootpath: "{{{{ ngi_pipeline_sthlm_delivery }}/nobackup/NGI/analysis_ready"
+    analysispath: <ROOTPATH>/ANALYSIS/<PROJECTID>
+    datapath: <ROOTPATH>/DATA/<PROJECTID>
+    stagingpath: <ROOTPATH>/DELIVERY/<PROJECTID>
+    deliverypath: /proj/<UPPNEXID>/INBOX/<PROJECTID>
+    report_aggregate: ngi_reports ign_aggregate_report -n stockholm --pandoc_binary
```

Comment:
And if two versions are needed then "stockholm" should probably be a variable that can be set to "uppsala" in the other case.